pkg/compiler/testdata/errors2.txt

```diff
@@ -144,17 +144,17 @@ foo$201(a ptr[in, s1])
 foo$202(a ptr[in, s7])
 foo$203(a u1)
 
-foo$204(a ptr[in, slen1])
+foo$204(a ptr[in, slen1], b ptr[in, no_slen22])
 
 slen1 {
 	f0 len[parent, int32]
-	f1 len[parent:f0, int32]	### parent can't be part of path expressions
-	f2 len[slen21:a, int32]		### len target a does not exist
-	f3 len[f0:parent, int32]	### parent can't be part of path expressions
+	f1 len[parent:f0, int32]
+	f2 len[slen21:a, int32]		### len target a does not exist in slen2, parents: <foo$204>, slen1
```

Comment:
How did "slen2" come up here? The parent name is "slen1", and the expression references "slen21". I am confused.

Reply:
`slen2` is the actual name of the struct where it was looking for the field named `a`, `slen21` was of type `slen2`